Build error (undefined reference to `smartcard_DVCPluginEntry')

cmake -DCMAKE_BUILD_TYPE=Debug -DWITH_SSE2=ON -DWITH_PCSC=ON -DWITH_MANPAGES=OFF .

Linking C shared library libfreerdp-gdi.so
[ 95%] Built target freerdp-gdi
[ 96%] [ 96%] [ 97%] Building C object client/X11/CMakeFiles/xfreerdp.dir/xf_gdi.c.o
Building C object client/X11/CMakeFiles/xfreerdp.dir/xf_rail.c.o
[ 97%] Building C object client/X11/CMakeFiles/xfreerdp.dir/xf_tsmf.c.o
Building C object client/X11/CMakeFiles/xfreerdp.dir/xf_event.c.o
[ 97%] Building C object client/X11/CMakeFiles/xfreerdp.dir/xf_cliprdr.c.o
[ 98%] Building C object client/X11/CMakeFiles/xfreerdp.dir/xf_monitor.c.o
[ 98%] Building C object client/X11/CMakeFiles/xfreerdp.dir/xf_graphics.c.o
[ 99%] Building C object client/X11/CMakeFiles/xfreerdp.dir/xf_keyboard.c.o
[ 99%] Building C object client/X11/CMakeFiles/xfreerdp.dir/xf_window.c.o
[100%] Building C object client/X11/CMakeFiles/xfreerdp.dir/xfreerdp.c.o
Linking C executable xfreerdp
../common/libfreerdp-client.so.1.1.0: undefined reference to `smartcard_DVCPluginEntry'
collect2: error: ld returned 1 exit status
make[2]: **\* [client/X11/xfreerdp] Error 1
make[1]: **\* [client/X11/CMakeFiles/xfreerdp.dir/all] Error 2
make: **\* [all] Error 2

--- a/channels/smartcard/client/CMakeLists.txt
+++ b/channels/smartcard/client/CMakeLists.txt
@@ -23,7 +23,7 @@ set(${MODULE_PREFIX}_SRCS
 
 include_directories(${PCSC_INCLUDE_DIRS})
 
-add_channel_client_library(${MODULE_PREFIX} ${MODULE_NAME} ${CHANNEL_NAME} TRUE "DVCPluginEntry")
+add_channel_client_library(${MODULE_PREFIX} ${MODULE_NAME} ${CHANNEL_NAME} TRUE "DeviceServiceEntry")
 
 set_target_properties(${MODULE_NAME} PROPERTIES PREFIX "")
 

